app/views/static_pages/how.html.erb

```diff
@@ -19,9 +19,16 @@
     </p>
     <br>
     <p>
-      Want to learn more or be a part of the she’s coding team? <%= link_to "Connect with us", "/connect" %>, or check out our project management systems: <%= link_to "Github", "https://github.com/shescoding", target: "_blank" %>, <%= link_to "Google Group", "https://groups.google.com/forum/#!forum/shescoding", target: "_blank" %>, <%= link_to "Trello", "https://trello.com/shescoding", target: "_blank" %>, <%= link_to "Slack", "https://shescoding.slack.com/", target: "_blank" %>.
+      Want to learn more or be a part of the she’s coding team? <%= link_to "Connect with us", "/connect" %>, or check out our project management systems: 
     </p>
   </div>
+  <div class="connect">
+     <ul class="connect_icons">
+      <li><%= link_to image_tag("static_pages/github_logo.png"), "https://github.com/shescoding",  title: 'Github', rel: 'tooltip', class: 'tooltips', target: "_blank" %><!-- <span>She's Coding on Github</span> --></li>
+      <li><%= link_to image_tag("static_pages/google_groups_logo.png"), "https://github.com/shescoding", title: 'Google Groups', rel: 'tooltip', target: "_blank" %></li>
+      <li><%= link_to image_tag("static_pages/trello_logo.png"), "https://github.com/shescoding", title: 'Trello', rel: 'tooltip', target: "_blank" %></li>
+      <li><%= link_to image_tag("static_pages/slack_logo.png"), "https://github.com/shescoding", title: 'Slack', rel: 'tooltip', target: "_blank" %></li>
```

Comment:
Currently all the images link to Github, whereas they should link to their specific site.